core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -52,8 +51,8 @@ object ZooKeeperClient {
  * @param sessionTimeoutMs session timeout in milliseconds
  * @param connectionTimeoutMs connection timeout in milliseconds
  * @param maxInFlightRequests maximum number of unacknowledged requests the client will send before blocking.
+ * @param clientConfig ZooKeeper client configuration, for TLS configs if desired
```

Comment:
Nice, I believe we might need to update the have java doc in `registerStateChangeHandler` and `unregisterStateChangeHandler` in same file as well

Reply:
To be honest, I considered not touching that file at all since we expect to delete it soon. I wouldn't spend too much time cleaning it up, so I'll ignore these other issues.